fuel/app/config/lti.php

```diff
@@ -1,6 +1,115 @@
+			// remote_identifier is the name of the lti data sent
+			// local_identifier is the name of the user object property that we will match the remote identifier against
+			// ex: incoming lis_person_sourceid = 'dave', we'll look for Model_User::query()->where($local_identifier, Input::post($remote_identifier))
+			// another option is to use email instead of sourcedid, remote = 'lis_person_contact_email_primary' and local = 'email'
+			'remote_identifier'     => $_ENV['LTI_REMOTE_IDENTIFIER'] ?? 'lis_person_sourcedid',
+			'local_identifier'      => 'username',
+
+			// When true, materia will accept user data from the external system.
+			// This means it will create users we don't have and update their user
+			// data if it changes. It will NOT update any external roles
+			// (see 'use_launch_roles')
+			'creates_users'         => $_ENV['BOOL_LTI_CREATE_USERS'] ?? true,
+
+			// allow an external system to define user roles in Materia
+			'use_launch_roles'      => $_ENV['BOOL_LTI_USE_LAUNCH_ROLES'] ?? true,
+
+			// which auth driver will do the final work authenticating this user
+			'auth_driver'           => 'LtiAuth',
+
+			// Should we bother saving the assocation of the chosen widget to the resource
+			// most LTI consumers do not actually know which widget they are requesting
+			// however materia allows the LTI consumer to send and optional message that can request a specific widget
+			'save_assoc'            => true,
+
+			// How many seconds should the oauth token be valid since created
+			'timeout'               => 3600,
+
+			// Define the privacy level this integration to the consumer
+			// public
+			'privacy'               => 'public',
+
+			// Define aspects of the course navigation link
+			// such as whether it is available at all, who can see it, and what text it displays
+			'course_nav_default'    => ($_ENV['BOOL_LTI_COURSE_NAV_DEFAULT'] ?? false) ? 'enabled' : 'disabled',
+			'course_nav_enabled'    => 'true',
+			'course_nav_text'       => 'Materia',
+			'course_nav_visibility' => 'members',
+
+			'tool_id'               => $_ENV['LTI_TOOL_ID'] ?? 'io.github.ucfopen',
+
+			// Canvas launches with `ext_outcome_data_values_accepted=text,url`
+			// this flag upgrades `url` support to `ltiLaunchUrl`
+			// I believe this is a custom Canvas convention.
+			// Disabled if the LTI consumer doesn't handle upgrading url to ltiLaunchUrl
+			'upgrade_to_launch_url' => true,
+
+			// Security Settings CHANGE SECRET AT LEAST!!!
+			'secret'                => $_ENV['LTI_SECRET'],
+			'key'                   => $_ENV['LTI_KEY'],
+
+			// When sending LTI ContentItem Selection requests, Sakai updated and required
+			// that GET params in the url aren't duplicated in the body of params
+			// as they caused oauth signature validation to fail in Sakai.
+			'enforce_unique_params' => $_ENV['LTI_ENFORCE_UNIQUE_PARAMS'] ?? true,
```

Comment:
I added this option as it's not super clear if other platforms will want this or not.